### ISSUE TITLE
Callback or event emitted on finish?

### ISSUE THREAD

**Issue by Paul-Ver** [alxhotel/chromecast-api#11]
Will the library emit an event when the video has ended?

If not, it would be a nice feature to be able to start the next video when the current one finishes.

**Comment by alxhotel** [comment]
Definitely. This would be a great feature.

Pull requests are welcome ;)

**Comment by Paul-Ver** [comment]
I don't yet have a chromecast, I've got one of those cheap chinese miracast devices.
But saw your NPM module for chromecast support, which I thought would be nice to implement in a home automation project.
And for that it'd be nice to also be able to play music, and the next file when the first one ends.

Once I really have time to get started with it (and come to the point of chromecast integration) I may try to help out ;)
But I was wondering if it would be possible, it hasn't got a high priority for me :)


I've checked out the code and found this in "device.js", I'm not all too known with the chromecast API and I can't test it myself.

But if there are statuses like "ready" and "playing", it'd be easy to implement.

```
player.on('status', function (status) {
        if (status) {
          debug('PlayerState = %s', status.playerState)
          self.emit('status', status)
        }
}
```

**Comment by alxhotel** [comment]
You can check out how Butterproject handles that case in [here](https://github.com/butterproject/butter-desktop/blob/master/src/app/lib/views/player/loading.js#L168-L188).

Basically, when you receive an `IDLE` state and you know you already have started the video, then you can say that the video has stopped.

**Comment by alxhotel** [comment]
Finally, this feature has been implemented in 05308bd985a9c2b47c670c9564b131d37eebc53a.

I'm planning to make a new release this week with this feature.

**Comment by Paul-Ver** [comment]
Good!
I haven't tested it yet; but I'll close this issue for now (looks cleaner), if I find any issues I may reopen it or send a pull request :)
